message_ix_models/tools/iea/weo.py

```diff
@@ -0,0 +1,672 @@
+"""
+Code for handling IEA WEO data
+"""
+
+import numpy as np
+import pandas as pd
+
+from message_ix_models.util import package_data_path
+
+
+def get_weo_data():
+    """
+    Read in raw WEO investment/capital costs and O&M costs data
+    (for all technologies and for STEPS scenario only).
+    Convert to long format
+
+    Returns DataFrame of processed data
+    """
+
+    # Read in raw data file
+    file_path = package_data_path(
+        "iea", "WEO_2022_PG_Assumptions_STEPSandNZE_Scenario.xlsb"
+    )
+
+    # Dict of all of the technologies,
+    # their respective sheet in the Excel file,
+    # and the start row
+    tech_rows = {
```

Comment:
All of these fixed values can be moved outside functions—even if they're only used in one function.

As globals, they can be named in all caps, e.g. `TECH_ROWS`.